Type/Password.pm

```diff
@@ -5,6 +5,33 @@ use Bivio::Base 'Type.Name';
 use Bivio::TypeError;
 use Digest::SHA ();
 
+my($_F) = b_use('IO.File');
+
+my($_WEAK_PASSWORDS);
+my($_C) = b_use('IO.Config');
+my($_CFG);
+$_C->register($_CFG = {
+    weak_regex => [],
+    weak_corpus_path => undef,
+    in_weak_corpus => sub {
+        # This implementation should only be used for a corpus of limited size. Larger corpuses
+        # should be stored in a more sofisticated external database, with a new implementation
```

Comment:
I don't think we need a sophisticated db. It can just be a bdb file, like the quote db. It's a single writer, multi reader problem. bdb is great for that. You can then build the bdb in Biz.File by pulling a file dynamically from a shellutil. The corpus could be manually built or as a db upgrade.